jmx-scraper/src/main/resources/wildfly.yaml

```diff
@@ -0,0 +1,104 @@
+---
+
+rules:
+  - bean: jboss.as:deployment=*,subsystem=undertow
+    metricAttribute:
+      deployment: param(deployment)
+    prefix: wildfly.session.
+    unit: "{session}"
+    mapping:
+      sessionsCreated:
+        metric: count
+        type: counter
+        desc: The number of sessions created.
+      activeSessions:
+        metric: active
+        type: updowncounter
+        desc: The number of currently active sessions.
+      expiredSessions:
+        metric: expired
+        type: counter
+        desc: The number of sessions that have expired.
+      rejectedSessions:
+        metric: rejected
+        type: counter
+        desc: The number of sessions that have been rejected.
+
+  - bean: jboss.as:subsystem=undertow,server=*,http-listener=*
+    metricAttribute:
+      server: param(server)
+      listener: param(http-listener)
+    prefix: wildfly.
+    type: counter
+    mapping:
+      requestCount:
+        metric: request.count
+        unit: "{request}"
+        desc: The number of requests received.
+      processingTime:
+        metric: request.time
+        unit: ns
+        desc: The total amount of time spent on requests.
+      errorCount:
+        metric: request.server_error
+        unit: "{request}"
+        desc: The number of requests that have resulted in a 5xx response.
+      bytesSent:
+        metric: network.io
+        unit: "by"
+        desc: "The number of bytes transmitted."
+        metricAttribute:
+          state: const(out)
+      bytesReceived:
+        metric: network.io
+        unit: "by"
+        desc: "The number of bytes transmitted."
+        metricAttribute:
+          state: const(in)
+
+  - bean: jboss.as:subsystem=datasources,data-source=*,statistics=pool
+    metricAttribute:
+      data_source: param(data-source)
+    type: counter
+    prefix: wildfly.jdbc.
+    mapping:
+      ActiveCount:
```

Comment:
ActiveCount and IdleCount could use YAML references since they share a lot. The same applies to few props in the bean below. What do you think about it?